### PR TITLE
📘  compare substring, substr and slice

### PR DESCRIPTION
**Pull request by skrzepij**

### CODE REVIEW

**Review by skrzepij — COMMENTED** [review]
Need to check how it looks on local dev build.

File: content/definitions/substring-substr-i-slice.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+console.log(text.substr(1, 5)) // 'ronte' - 5 znaków zaczynając od drugiego indeksu.
+console.log(text.substr(-3, -1)) // '' -  długość nie może być ujemna
+console.log(text.substr(8, 5)) // '' - nie można podawać zakresu w odwrotnej kolejności
+```
+
+- `Slice` Przyjmuje dwa parametry. Tak samo jak `substring` - początkowy oraz końcowy indeks.
+
+```javascript
+const text = 'Frontend'
+console.log(text.substr(0)) // 'Frontend' - zwracamy całą wartość
```
Comment (MateuszWasik) [inline content/definitions/substring-substr-i-slice.md]: instead `substr` should be `slice` I guess. Cabbage head

File: content/definitions/substring-substr-i-slice.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+console.log(text.substr(-3)) // 'end' - pierwszy parametr może być ujemny
+console.log(text.substring(1, 5)) // 'ront' - zakres zaczynamy od drugiego i kończymy na piątym indeksie
+console.log(dupa.slice(-3, -1)) // 'en' - można bazować na ujemnych indeksach
```
Comment (MateuszWasik) [inline content/definitions/substring-substr-i-slice.md]: Dupa is not very nice. You should use `ass` or something else. Cabbage head

**Review by MateuszWasik — APPROVED** [review]

File: content/definitions/substring-substr-i-slice.md (hunk 1 above)
Comment (skrzepij) [reply content/definitions/substring-substr-i-slice.md]: image

**Pull request merged**